strapi/strapi
# Fixed Bug: Markdown Editor Auto-scrolls to top when formatting applied #20160

What does it do?
This pull request fixes an issue where the Markdown Editor auto-scrolls to the top when applying formatting (e.g., bullet lists, headings). The editor's position is maintained during formatting operations, improving the editing experience.

Why is it needed?
The auto-scrolling issue disrupts the user's workflow by jumping to the top of the page when formatting is applied. This fix ensures that the editor maintains its position, allowing users to continue editing without unnecessary scrolling.

How to test it?
1.Open the Markdown Editor in the application.
2.Create or open a document with some content.
3.Apply different types of formatting (e.g., bullet lists, headings) to the content.
4.Verify that the editor does not scroll to the top during or after applying the formatting.

Related issue(s)/PR(s)
Fixes https://github.com/strapi/strapi/issues/20160.

Let us know if this is related to any issue/pull request
